src/main/java/org/cloudburstmc/server/Server.java

```diff
@@ -365,8 +365,8 @@ public void boot() throws IOException {
                     "locale/cloudburst/configs");
             configLocaleManager.setLocale(locale);
 
-            File configFile = new File(this.dataPath + "nukkit.yml");
-            InputStream stream = Nukkit.class.getClassLoader().getResourceAsStream("nukkit.yml");
+            File configFile = new File(this.dataPath + "cloudburst.yml");
```

Comment:
Before we rename this, we need to change the filenames in the language submodule?